### ISSUE TITLE
[Proposal] Admin-LTE integration

### ISSUE THREAD

**Issue by ThangLuuQuoc** [amranidev/scaffold-interface#94]
I think scaffold-interface is awesome, but it miss some important features.
I gave it a trying, and I easy to generate all file for my module with full layout and ready to use.
I use result as init front-end template in my app, and I must take more time to clone it to current AdmindLTE to manage generated module that I just created. 
and finally, I hope add a option to generate admin management for the generated table same with current (Permissions, Roles, Users) and current file will use as front-end

**Comment by amranidev** [comment]
@ThangLuuQuoc 
do you mean that (model/entity) layouts must handle AdminLTE as well as dashboard ??

**Comment by ThangLuuQuoc** [comment]
yes

**Comment by amranidev** [comment]
@ThangLuuQuoc 

This feature is available now in dev-master.

```json
require : {
   "Amranidev/scaffold-interface": "dev-master"
}
```
Feel free to try it out and submit any issue or suggestion, before it would be released :)

**Comment by amranidev** [comment]
@ThangLuuQuoc 
scaffold-interface is released now with ([v1.6.31](https://github.com/amranidev/scaffold-interface/blob/master/CHANGELOG.md))
